### PR TITLE
Tenancies: publicinfo

### PR DESCRIPTION
**Pull request by eccles**
Add access to the publicinfo of the tenencies endpoint.

Signed-off-by: Paul Hewlett <phewlett76@gmail.com>

### CODE REVIEW

File: examples/get_publicasset.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
 """Get a public asset from a instance of Archivist
 """
 
+from warnings import filterwarnings
+
 from archivist.archivistpublic import ArchivistPublic
 
+filterwarnings("ignore", message="Unverified HTTPS request")
```
Comment (Serhiy1) [inline examples/get_publicasset.py:8]: Quick question: what is the purpose of the filter?

Comment (eccles) [reply examples/get_publicasset.py:8]: the urllib library uses the 'warning' package to do logging about doing unverified requests - if this is not set than stdout is littered with warnings which makes things noisy

**Review by Serhiy1 — APPROVED** [review]

**Review by honourfish — CHANGES_REQUESTED** [review]

File: archivist/tenancies.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+        return identity
+
+    def publicinfo(self, identity: str) -> Tenant:
+        """Read Tenant public info
+
+        Reads Tenant public info
+
+        Args:
+            identity (str): tenancies identity e.g. tenancies/xxxxxxxxxxxxxxxxxxxxxxx
```
[the remark this answers is not in the document]
Comment (eccles) [reply archivist/tenancies.py:78]: well no - the user will expect to push the identity directly received from another endpoint - this logic hides the fact that this is currently incorrectly implemented

[the remark this answers is not in the document]
Comment (eccles) [reply archivist/tenancies.py:78]: I will add comment

**Pull request merged**